src/components/ItaliaTheme/Blocks/VideoGallery/Block/ViewBlock.jsx

```diff
@@ -134,10 +139,18 @@ const ViewBlock = ({ data, index, isEditMode = false }) => {
                     type="video/mp4"
                   />
                 ) : data.allowExternals ? (
-                  <Embed url={data.url} {...embedSettings} />
+                  <>
+                    <Embed url={data.url} {...embedSettings} />
+                    {showVideoCaption && (
+                      <p className="px-3 mt-2 figure-caption">{data.title}</p>
+                    )}
+                  </>
```

Comment:
```suggestion
                    <Embed url={data.url} {...embedSettings} />                    
```

ok, ho capito cosa intendevi